Lib/sqlite3/test/userfunctions.py

```diff
@@ -351,6 +298,24 @@ def test_empty_blob(self):
         cur = self.con.execute("select isblob(x'')")
         self.assertTrue(cur.fetchone()[0])
 
+    def test_func_params(self):
+        self.con.create_function("boomerang", 1, lambda x: x)
+        dataset = (
+            (42, int),
+            (1<<42, int),  # long long
+            (3.14, float),
+            ("text", str),
+            ("1\x002", str),
+            (b"blob", bytes),
```

Comment:
Add tests for bytearray and memoryview.

It would be nice to add also test for non-continuous memoryview (`memoryview(...)[::2]`).

Reply:
For non-contiguous buffers, we currently overwrite the (correct) `BufferError` with `TypeError`:

`ValueError: could not convert BLOB to buffer`

instead of:

`BufferError: memoryview: underlying buffer is not C-contiguous`

I'm leaning towards _not_ overwriting the exception, and raise `BufferError`, since it is more accurate.